cr-interactors/src/test/java/com/thefuntasty/mvvm/crinteractors/testinteractors/base/BaseCoroutineScopeOwnerTest.kt

```diff
@@ -0,0 +1,27 @@
+package com.thefuntasty.mvvm.crinteractors.testinteractors.base
+
+import com.thefuntasty.mvvm.crinteractors.CoroutineScopeOwner
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.test.TestCoroutineDispatcher
+import kotlinx.coroutines.test.TestCoroutineScope
+import kotlinx.coroutines.test.setMain
+import org.junit.After
+import org.junit.Before
+
+abstract class BaseCoroutineScopeOwnerTest : CoroutineScopeOwner {
```

Comment:
What's the purpose of this test?

Reply:
it's base class which handles coroutineScope and dispatcher setup